style.css

```diff
@@ -0,0 +1,198 @@
+body {
```

Comment:
As a general comment and improvement suggestion: 
I see some elements overflowing their containers when the screen is resized, which is quite common and often requires adjustments. 
My recommendation overall is to use **relative length units** ([Link](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units)). 

In my case, my favorite is "_em_," but there are others like _%_ or _rem_. It's a matter of finding one that you're familiar with and practical for you. 

Another suggestion is to use **Chrome Developer Tools** [Link](https://developer.chrome.com/docs/devtools/overview) ,where you can modify values and see them immediately on your screen, and then, when you're satisfied, implement them in your code.

If you have any doubts about this, feel free to contact me and we can work on it together. 🤝

Practice makes perfect, and the more challenges you encounter, the more you'll grow on this path to becoming a web developer. 🚀🚀